R/object_usage_linter.R

```diff
@@ -280,10 +291,19 @@ get_imported_symbols <- function(xml) {
   import_exprs <- xml_find_all(xml, import_exprs_xpath)
   imported_pkgs <- get_r_string(import_exprs)
 
-  unlist(lapply(imported_pkgs, function(pkg) {
+  unlist(lapply(seq_along(import_exprs), function(i) {
     tryCatch(
-      getNamespaceExports(pkg),
-      error = function(e) character()
+      getNamespaceExports(imported_pkgs[[i]]),
+      error = function(e) {
+        pkg <- imported_pkgs[[i]]
+        lint_node <- xml2::xml_parent(import_exprs[[i]])
+        # nolint start: undesirable_function_name. .libPaths() is neccessary here.
+        lint_msg <- paste0("Could not find exported symbols for package \"", pkg, "\" in library ",
+                           toString(shQuote(.libPaths())), ". This may lead to false positives.")
```

Comment:
The usual message will be "there is no package called ‘blah’".
Showing .libPaths() to help show where the package should be.